libbeat/docs/dashboards.asciidoc

```diff
@@ -9,6 +9,15 @@
 //// include::../../libbeat/docs/dashboards.asciidoc[]
 //////////////////////////////////////////////////////////////////////////
 
+ifdef::has_solutions[]
+TIP: The example dashboards are a great way to get started visualizing
```

Comment:
This seems a bit long to me. I'm wondering if we can take out some of the words. Maybe even get rid of the first sentence?

> TIP: ~The example dashboards are a great way to get started visualizing {beatname_uc} data in {kib}.~ For deeper observability into your infrastructure, use the {infra-guide}/infrastructure-ui-overview.html[Infrastructure] and {infra-guide}/logs-ui-overview.html[Logs] UIs in {kib}. For setup details, see the {infra-guide}/index.html[Infrastructure Monitoring Guide].